apps/studio/src/features/sign-in/components/SingpassLogin/SingpassLoginButton.tsx

```diff
@@ -23,24 +27,27 @@ export const SingpassLoginButton = (): JSX.Element | null => {
   }
 
   return (
-    <Stack gap="0.75rem">
-      <Button
-        colorScheme="neutral"
-        height="2.75rem"
-        size="xs"
-        variant="outline"
-        isLoading={singpassLoginMutation.isLoading}
-        onClick={handleSingpassLogin}
-        aria-label="Authenticate with Singpass"
-      >
-        <Flex align="center" flexDirection="row" aria-hidden>
-          <Text>Authenticate with </Text>
-          {/* Negative margin so the svg sits on the same line as the text */}
-          <Box mb="-3px">
-            <SingpassFullLogo height="1rem" />
-          </Box>
-        </Flex>
-      </Button>
-    </Stack>
+    <Button
+      height="2.75rem"
+      width="full"
+      size="sm"
+      variant="outline"
+      colorScheme="clear"
+      bgColor="#F4333D"
+      _disabled={{ bgColor: "#F4333D" }}
+      _hover={{ bgColor: "#B0262D" }}
```

Comment:
nit: should be using colour tokens here

Reply:
Hmm these are custom colours used only on the Singpass login button (the bright red and dark red on hover), hence didn't create new tokens since it is quite unlikely it would be used elsewhere.